<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="3" ignored="0" total="3" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="25060" started-at="2023-08-25T18:44:27 IST" finished-at="2023-08-25T18:44:52 IST">
    <groups>
    </groups>
    <test name="ChromeTest" duration-ms="20476" started-at="2023-08-25T18:44:27 IST" finished-at="2023-08-25T18:44:47 IST">
      <class name="TestCases.TC5_ShopProduct">
        <test-method status="PASS" signature="LaunchApplication(java.lang.String)[pri:0, instance:TestCases.TC5_ShopProduct@50a638b5]" name="LaunchApplication" is-config="true" duration-ms="7446" started-at="2023-08-25T18:44:27 IST" finished-at="2023-08-25T18:44:34 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LaunchApplication -->
        <test-method status="PASS" signature="setupDriver()[pri:0, instance:TestCases.TC5_ShopProduct@50a638b5]" name="setupDriver" is-config="true" duration-ms="1" started-at="2023-08-25T18:44:34 IST" finished-at="2023-08-25T18:44:34 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupDriver -->
        <test-method status="FAIL" signature="ShopProduct()[pri:0, instance:TestCases.TC5_ShopProduct@50a638b5]" name="ShopProduct" duration-ms="12172" started-at="2023-08-25T18:44:34 IST" finished-at="2023-08-25T18:44:46 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Step1 - User is NOT logged in successfully expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Step1 - User is NOT logged in successfully expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at TestCases.TC5_ShopProduct.ShopProduct(TC5_ShopProduct.java:53)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ShopProduct -->
        <test-method status="PASS" signature="CloseApplication()[pri:0, instance:TestCases.TC5_ShopProduct@50a638b5]" name="CloseApplication" is-config="true" duration-ms="678" started-at="2023-08-25T18:44:46 IST" finished-at="2023-08-25T18:44:47 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CloseApplication -->
      </class> <!-- TestCases.TC5_ShopProduct -->
    </test> <!-- ChromeTest -->
    <test name="EdgeTest" duration-ms="21030" started-at="2023-08-25T18:44:27 IST" finished-at="2023-08-25T18:44:48 IST">
      <class name="TestCases.TC5_ShopProduct">
        <test-method status="PASS" signature="LaunchApplication(java.lang.String)[pri:0, instance:TestCases.TC5_ShopProduct@6a192cfe]" name="LaunchApplication" is-config="true" duration-ms="8295" started-at="2023-08-25T18:44:27 IST" finished-at="2023-08-25T18:44:35 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[Edge]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LaunchApplication -->
        <test-method status="PASS" signature="setupDriver()[pri:0, instance:TestCases.TC5_ShopProduct@6a192cfe]" name="setupDriver" is-config="true" duration-ms="1" started-at="2023-08-25T18:44:35 IST" finished-at="2023-08-25T18:44:35 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupDriver -->
        <test-method status="FAIL" signature="ShopProduct()[pri:0, instance:TestCases.TC5_ShopProduct@6a192cfe]" name="ShopProduct" duration-ms="11801" started-at="2023-08-25T18:44:35 IST" finished-at="2023-08-25T18:44:47 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Step1 - User is NOT logged in successfully expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Step1 - User is NOT logged in successfully expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at TestCases.TC5_ShopProduct.ShopProduct(TC5_ShopProduct.java:53)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ShopProduct -->
        <test-method status="PASS" signature="CloseApplication()[pri:0, instance:TestCases.TC5_ShopProduct@6a192cfe]" name="CloseApplication" is-config="true" duration-ms="764" started-at="2023-08-25T18:44:47 IST" finished-at="2023-08-25T18:44:48 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CloseApplication -->
      </class> <!-- TestCases.TC5_ShopProduct -->
    </test> <!-- EdgeTest -->
    <test name="FirefoxTest" duration-ms="25060" started-at="2023-08-25T18:44:27 IST" finished-at="2023-08-25T18:44:52 IST">
      <class name="TestCases.TC5_ShopProduct">
        <test-method status="PASS" signature="LaunchApplication(java.lang.String)[pri:0, instance:TestCases.TC5_ShopProduct@3835c46]" name="LaunchApplication" is-config="true" duration-ms="12876" started-at="2023-08-25T18:44:27 IST" finished-at="2023-08-25T18:44:39 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[Firefox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LaunchApplication -->
        <test-method status="FAIL" signature="ShopProduct()[pri:0, instance:TestCases.TC5_ShopProduct@3835c46]" name="ShopProduct" duration-ms="11441" started-at="2023-08-25T18:44:39 IST" finished-at="2023-08-25T18:44:51 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Step1 - User is NOT logged in successfully expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Step1 - User is NOT logged in successfully expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at TestCases.TC5_ShopProduct.ShopProduct(TC5_ShopProduct.java:53)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ShopProduct -->
        <test-method status="PASS" signature="setupDriver()[pri:0, instance:TestCases.TC5_ShopProduct@3835c46]" name="setupDriver" is-config="true" duration-ms="0" started-at="2023-08-25T18:44:39 IST" finished-at="2023-08-25T18:44:39 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupDriver -->
        <test-method status="PASS" signature="CloseApplication()[pri:0, instance:TestCases.TC5_ShopProduct@3835c46]" name="CloseApplication" is-config="true" duration-ms="655" started-at="2023-08-25T18:44:51 IST" finished-at="2023-08-25T18:44:52 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CloseApplication -->
      </class> <!-- TestCases.TC5_ShopProduct -->
    </test> <!-- FirefoxTest -->
  </suite> <!-- Suite -->
</testng-results>
